osc/gitea_api/pr.py

```diff
@@ -174,6 +174,35 @@ def yes_no(value):
 
         return str(table)
 
+    def dict(self, exclude_columns: Optional[list] = None):
```

Comment:
pylint now throws an error that is most likely related to this:
```
osc/gitea_api/pr.py:14:29: E0601: Using variable 'dict' before assignment (used-before-assignment)
```

I think it's a false positive, but we should take a closer look.

Reply:
Yes I am certain that it is false positive. As workaround we can rename this method (`to_dict` ?) or e.g. somehow do not use `data: dict` in __init__ method early in the file